src/flood_forecaster/data_model/river_station.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, DateTime
```

Comment:
IMHO `river_station.py` is not descriptive enough. I mean this file contains class to represent the tables so maybe a name like `river_level_tables.py`.

Reply:
When working with ORMs in a project, the data model or using the [Repository pattern](https://dddinpython.hashnode.dev/implementing-the-repository-pattern-using-sqlalchemy) you abstract the database (and the fact that there are tables) from the business domain. Sadly there's no much time to do it as proper as I'd like to but that's the main reason for the naming.